### DISCUSSION TITLE
[Enhancement]: More customizability to the Cider's Inbuilt Discord Rich Presence

### DISCUSSION THREAD

**Discussion by Monochromish** [ciderapp/Cider#655] [Added Features]
### The feature you would like

**More customizability to the Cider's Inbuilt Discord Rich Presence**

Options to either enable or disable elapsed time/buttons
Able to change the format of displaying currently playing music

### Alternatives and Other Information

null

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Amaru8** [comment]
These were implemented some time ago, locking discussion.
